app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  before_action :login_confirm
```

Comment:
As mentioned above Devise provides this with `authenticate_user` which can be used.